core/src/ast/function.rs

```diff
@@ -351,6 +352,15 @@ impl ToSql for Function {
                 format!("UNWRAP({}, {})", expr.to_sql(), selector.to_sql())
             }
             Function::GenerateUuid() => "GENERATE_UUID()".to_owned(),
+            Function::Greatest(items) => {
+                let items = items
+                    .iter()
+                    .filter(|&item| !matches!(item, Expr::IsNull(_)))
```

Comment:
Oh, I missed the this one.
we don't need to filter out the `NULL` arguments. `NULL` also needs to be showed.
```
GREATEST(NULL, 9, 7)
GREATEST(1, 2, 3)
GREATEST(NULL, NULL, 2)
```
all above expressions are valid.